docs/core/tools/dotnet-new.md

```diff
@@ -482,24 +708,28 @@ Each project template may have additional options available. The core templates
 
 **console, xunit, mstest, web, webapi**
 
-`-f|--framework` - Specifies the [framework](../../standard/frameworks.md) to target. Values: `netcoreapp1.0` or `netcoreapp1.1`. The default value is `netcoreapp1.0`.
+`-f|--framework <FRAMEWORK>` - Specifies the [framework](../../standard/frameworks.md) to target. Values: `netcoreapp1.0` or `netcoreapp1.1`. The default value is `netcoreapp1.0`.
 
 **classlib**
 
-`-f|--framework` - Specifies the [framework](../../standard/frameworks.md) to target. Values: `netcoreapp1.0`, `netcoreapp1.1`, or `netstandard1.0` to `netstandard1.6`. The default value is `netstandard1.4`.
+`-f|--framework <FRAMEWORK>` - Specifies the [framework](../../standard/frameworks.md) to target. Values: `netcoreapp1.0`, `netcoreapp1.1`, or `netstandard1.0` to `netstandard1.6`. The default value is `netstandard1.4`.
 
 **mvc**
 
-`-f|--framework` - Specifies the [framework](../../standard/frameworks.md) to target. Values: `netcoreapp1.0` or `netcoreapp1.1`. The default value is `netcoreapp1.0`.
+`-f|--framework <FRAMEWORK>` - Specifies the [framework](../../standard/frameworks.md) to target. Values: `netcoreapp1.0` or `netcoreapp1.1`. The default value is `netcoreapp1.0`.
 
-`-au|--auth` - The type of authentication to use. Values: `None` or `Individual`. The default value is `None`.
+`-au|--auth <AUTHENTICATION_TYPE>` - The type of authentication to use. Values: `None` or `Individual`. The default value is `None`.
 
 `-uld|--use-local-db` - Specifies whether or not to use LocalDB instead of SQLite. Values: `true` or `false`. The default value is `false`.
 
 ---
 
 ## Examples
 
+Create a C# console application project:
+
+`dotnet new "Console Application"`
```

Comment:
I wouldn't expect this to be the standard way of creating C# console apps. Do we need to mention the match feature here as well?